draw/src/test/resources/robolectric.properties

```diff
@@ -0,0 +1 @@
+sdk=33
```

Comment:
Ah I meant to leave a note about this. Basically `targetSdk` (in `build.gradle`) is now deprecated. I think the idea is that it doesn't really make sense given you already have a `minSdk` and a `compileSdk` (which I don't disagree with). The problem is that Robolectric automatically uses target SDK so if it isn't configured it uses the min SDK. Rather than use the deprecated Gradle config, it made sense to me to use the Robolectric specific config.

Do you think I should change this for all the modules that use `build.gradle.kts` (it doesn't seem to be deprecated in Groovy yet)?

Reply:
> Do you think I should change this for all the modules that use build.gradle.kts (it doesn't seem to be deprecated in Groovy yet)?

I think you can file a separate issue for that.